### PR TITLE
Flash cyber weekend sale

### PR DESCRIPTION
**Pull request by tholex**

### CODE REVIEW

File: demo/__env.json [hunk 1]
```diff
@@ -149,14 +149,5 @@
...
     "hotload": true
-  },
-  "user": {
-    "tf_login": "tholex@gmail.com",
-    "name": "Abe Z",
-    "first_name": "Abe",
-    "last_name": "X",
-    "id": 1,
-    "contact_id": 1,
-    "access": ["navbar-notifications"]
```
Comment (diezguerra) [inline demo/__env.json]: are you sure this doesn't break other things?

File: src/AppBar/Navigation.jsx [hunk 2]
```diff
@@ -64,6 +65,12 @@ class AppNav extends React.Component {
         }, 400);
     }
 
+    renderFlash() {
+        return (<div href="//www.thinkful.com/enroll/" className="app-nav-flash">
```
Comment (learykara) [inline src/AppBar/Navigation.jsx]: `config.enroll.url`?

File: src/AppBar/Navigation.jsx [hunk 3]
```diff
@@ -202,6 +209,14 @@ class AppNav extends React.Component {
         const navClassName = cx(
             'app-nav', {'app-nav__visible': this.state.isMenuVisible});
 
+        const blackFriday = moment("2015-11-27T00:00:00Z");
+        const cyberTuesday = moment("2015-12-01T12:00:00Z");
+        const current = moment();
```
Comment (learykara) [inline src/AppBar/Navigation.jsx]: `moment.tz('UTC')`

File: src/AppBar/Navigation.jsx [hunk 4]
```diff
@@ -202,6 +209,14 @@ class AppNav extends React.Component {
...
+        /* flashActive is true between Nov 27, 00:00 GMT, and
```
Comment (learykara) [inline src/AppBar/Navigation.jsx]: shouldn't it be true during those times in the user's timezone?

File: demo/__env.json (hunk 1 above)
Comment (tholex) [inline demo/__env.json]: reverted to keep demo page as-is

File: src/AppBar/Navigation.jsx (hunk 2 above)
Comment (tholex) [inline src/AppBar/Navigation.jsx]: turns out it's just not going to be linked, forgot to remove the href.

File: src/AppBar/Navigation.jsx (hunk 4 above)
Comment (tholex) [inline src/AppBar/Navigation.jsx]: Pretty sure this is OK? We'll start the sale when the UK hits black friday (7pm EST on Thanksgiving) and end it when japan hits midnight onDec 1st (~7am EST Dec 1st)? Asking justin

**Pull request merged**